packaging/vagrant/Vagrantfile

```diff
@@ -171,7 +171,32 @@ EOF2
     (cd shadow-4.6 && ./configure && sudo make install)
 
     # Install Buildah.
-    sudo yum -y install buildah
+    sudo yum -y install \
+    make \
+    golang \
```

Comment:
One gotcha is that the upstream Buildah we'll be switching to shortly requires Go 1.12. I assume that what `yum` is giving us is probably not that new.

Is it worth including the right Go in this PR? Looking briefly at the [installation instructions](https://golang.org/doc/install), it looks pretty straightforward.